firmware/src/main.cpp

```diff
@@ -70,22 +118,34 @@ HarpCApp& app = HarpCApp::init(who_am_i, hw_version_major, hw_version_minor,
                                harp_version_major, harp_version_minor,
                                fw_version_major, fw_version_minor,
                                serial_number, "Env Sensor",
```

Comment:
@bruno-f-cruz do you have a preferred name for this?

Reply:
EnvironmentSensor to match the repo and the name of the registered device would be awesome.